### DISCUSSION TITLE
Failed to build image

### DISCUSSION THREAD

**Discussion by op-family** [containerd/nerdctl#2251] [Q&A]
![图片](https://github.com/containerd/nerdctl/assets/47775316/46ee7b49-2428-4a04-a3fc-e65811f5ac2f)
I have installed the complete set of nerdctl and buildctl components in the container, but it failed to build using nerdctl. However, it can be built on the real machine, but it cannot be built once it is in the container. I hope everyone can help me

**Comment by yuchanns** [comment]
Ensure you add the flag `--privileged` with `run` command
